### DISCUSSION TITLE
How do we change the language to Japanese in IOS?

### DISCUSSION THREAD

**Discussion by jainilparikh** [appium/appium#16974] [Q&A] [answered]
Hi, 

I was able to change the keyboard layout to German using the following command:

`print(driver.execute_script("mobile: configureLocalization", {"keyboard": {"name": "de_CH", "layout": "QWERTZ", "hardware": "Automated"}}));`

How so I change the keyboard language to JA? Any suggestions?

**Comment by KazuCocoa** [comment, marked as answer]
Perhaps it is locale id format in https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html
e.g. ja_JP

**Reply by jainilparikh** [reply]
Thanks! this works
